### ISSUE TITLE
[Performance] Confirm the performance is not impacted after adding new WCAG 2.2 rule: target_spacing_sufficient

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1732]
The new WCAG 2.2 rule: **_target_spacing_sufficient_** checks each and every possible target on a web page to determine if it violates the new 2.5.8 Target Size minimum requirement, which _might impact_ the perceived performance of the Checker, especially on web site pages that have lots of targets (links, buttons, etc.).

### Test recommendations
BEFORE (release 3.1.63) and AFTER the release 3.1.64? that includes the new rule, **test** the TIME it takes to scan the following pages:
1. [CNN.com](https://www.cnn.com/) 330+ tab stops
2. [UTexas.edu]() 50+ tab stops 

- [ ] test performance
- [ ] **_If_** performance is noticeably slower, propose a solution, such as add documentation to note that the new rule is only run when WCAG 2.2 is selected, add note to rule help, add note to users guide about "optimizing for performance", etc.

**Comment by shunguoy** [comment]
close this as it works as expected.  In the future, if a performance issue related to the rule is identified, we will fix it on a case-by-case basis with a concrete test page where the issue was found.
